CONTRIBUTING.md

```diff
@@ -1,159 +1,307 @@
-# Contributing to the CRS
+# Contribution Guidelines
 
-We value third-party contributions. To keep things simple for you and us,
-please adhere to the following contributing guidelines.
+**Note:** An HTML version of these guidelines can be found on the CRS documentation site at https://coreruleset.org/docs/development/contribution_guidelines/
 
-## Getting Started
+> The CRS project values third party contributions. To make the contribution process as easy as possible, a helpful set of contribution guidelines are in place which all contributors and developers are asked to adhere to.
 
-* You will need a [GitHub account](https://github.com/signup/free).
-* Submit a [ticket for your issue](https://github.com/coreruleset/coreruleset/issues), assuming one does not already exist.
-  * Clearly describe the issue including steps to reproduce when it is a bug.
-  * Make sure you specify the version that you know has the issue.
-  * Bonus points for submitting a failing test along with the ticket.
-* If you don't have push access, fork the repository on GitHub.
+## Getting Started with a New Contribution
+
+1. Sign in to [GitHub](https://github.com/join).
+2. Open a [new issue](https://github.com/coreruleset/coreruleset/issues) for the contribution, *assuming a similar issue doesn't already exist*.
+    * **Clearly describe the issue**, including steps to reproduce if reporting a bug.
+    * **Specify the CRS version in question** if reporting a bug.
+    * Bonus points for submitting tests along with the issue.
+3. Fork the repository on GitHub and begin making changes there.
+4. Signed commits are preferred. (For more information and help with this, refer to the [GitHub documentation](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)).
 
 ## Making Changes
 
-* Please base your changes on the latest dev branch (e.g. ```v3.4/dev```)
-* Create a topic branch for your feature or bug fix.
-* Please fix only one problem at a time; this will help to quickly test and merge your change. If you intend to fix multiple unrelated problems, please use a separate branch for each problem.
+* Base any changes on the latest dev branch (e.g., `v3.4/dev`).
```

Comment:
The idea was to have this file auto-generated from the online version, but I don't know how to do that… (maybe a GitHub action or something?)

I'll remove the explicit version number for the time being.

Reply:
Removed.

Reply:
We have a release procedure https://github.com/coreruleset/coreruleset/wiki/Release-Procedure and updating version numbers all over the place is part of that, it could be added there.

Reply:
Good thinking!

Branch version now added back in.
New instruction added to the release procedure document.